.vscode/launch.json

```diff
@@ -117,6 +117,25 @@
       },
       "cwd": "${workspaceFolder}/packages/xml-views-validation"
     },
+    {
+      "type": "node",
+      "request": "launch",
+      "name": "XML completion: Run Current Jest File",
+      "program": "${workspaceFolder}/node_modules/.bin/jest",
+      "args": [
+        "${fileBasenameNoExtension}",
+        "--coverage=false",
+        "--config",
```

Comment:
We probably should start the tests only with one worker here as well. (pass `--maxWorkers=1`)

Reply:
done